parcels/grid.py

```diff
@@ -3,12 +3,13 @@
 from ctypes import Structure, c_int, c_float, c_double, POINTER, cast, c_void_p, pointer
 from enum import IntEnum
 
-__all__ = ['GridCode', 'StructuredGrid', 'GridIndex', 'CGrid']
+__all__ = ['GridCode', 'RectilinearGrid', 'RectilinearSGrid', 'GridIndex', 'CGrid']
```

Comment:
Call this `RectilinearZGrid` and RectilinearSGrid`, to highlight that one is not a subset or child of the other?